docs/source/deploying.rst

```diff
@@ -1,12 +1,43 @@
 .. Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 .. SPDX-License-Identifier: Apache-2.0
 
-Deploying to Daml Ledgers
-=========================
+Overview of Daml ledgers
+========================
+
+The following table lists commercially supported Daml ledgers and environments that are available
+today.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Product
+     - Synchronization Technology
+     - Status
+     - Canton Domain
+   * - Daml driver for VMware Blockchain
+     - VMware Blockchain
+     - GA
+     - No
+   * - Daml driver for PostgreSQL
+     - PostgreSQL
+     - GA
+     - Yes
+   * - Daml driver for Oracle DB
+     - Oracle DB
+     - GA
+     - Yes
+   * - Daml driver for Hyperledger Fabric
+     - Hyperledger Fabric
+     - Beta
+     - Yes
+   * - Daml driver for Hyperledger Besu
+     - Hyperledger Besu
+     - Beta
+     - Yes
 
 .. toctree::
   :maxdepth: 2
+  :hidden:
 
-  deploy/index
   deploy/generic_ledger
-  deploy/ledger-topologies
\ No newline at end of file
+  deploy/ledger_topologies
```

Comment:
why did you change this?

Reply:
David is removing in a different PR

Reply:
David is removing ledger_topologies in a different PR. I removed deploy/index to have less pages now that content is so minimal. I added the table to deploying.rst, the "actual" index that we use for assembly